ui/app/AppLayouts/Profile/views/SyncingView.qml

```diff
@@ -217,7 +217,7 @@ SettingsContentBase {
                             Layout.fillWidth: true
                             objectName: "enableMessageSyncingCheckBox"
 
-                            text: qsTr("Enable message syncing")
+                            text: qsTr("Restore my past messages to this device")
```

Comment:
Think we should have a "string freeze" policy in order; after the release branch is cut, no such changes allowed :) It will fix the string for one language (English) but break for N others. The translations will get broken for these and will need a re-review